catalog-rest-service/src/main/java/org/openmetadata/catalog/util/TablesInitializer.java

```diff
@@ -289,16 +291,16 @@ private static void usage() {
 
   private static void printToConsoleInDebug(String message) {
     if (DEBUG_MODE) {
-      System.out.println(message);
+      LOG.debug(message);
```

Comment:
these should be System.out only since logging is disabled using Logback.xml , nd Information needs to go to console

Reply:
@mohitdeuex, addressed in the latest commit.